content/2. Ondersteunende-informatie/4. Techniek/Git/3. Branches.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 3. Branches
```

Comment:
Valt me nu pas op, maar misschien handig om de naamgeving van de titel consistent te houden met het onderwerp uit de excel. dus in dit geval moet het worden `3. Git-Branches` ipv `3. Branches`. Geldt natuurlijk ook voor de andere bestanden

Reply:
Bespreekpunt 3